test/sdf/material_pbr.sdf

```diff
@@ -35,6 +36,7 @@
               <environment_map>environment_map.png</environment_map>
               <ambient_occlusion_map>ambient_occlusion_map.png</ambient_occlusion_map>
               <emissive_map>emissive_map.png</emissive_map>
+              <light_map uv_set="2u">light_map.png</light_map>
```

Comment:
Just curious, does it still work if we omit the `u`? Maybe we can test it?

Reply:
yep removed `u` in 45f5cb2 and test still passes